docs/Parallel-Building.md

```diff
@@ -1,6 +1,6 @@
-Theos is recommended to be used with the latest version of [GNU Make](https://www.gnu.org/software/make/). Unfortunately, the Xcode toolchain does not include the latest version of Make. In fact, it’s Make 3.81, from 2006!
+For the best results, Theos recommends that you use the latest version of [GNU Make](https://www.gnu.org/software/make/). Unfortunately, the Xcode toolchain does not include the latest version of Make. In fact, it includes Make 3.81 from 2006!
 
-While the Theos maintainers make their best efforts to support Make 3.81, there is a certain limit to what we can do. One of these limits is with parallel building. Parallel building is where a build tool like Make divides its work up in order to fully take advantage of all logical cores (threads) of the CPU. This is one significant way a build can be sped up, especially for larger projects.
+While the Theos maintainers do their best to support Make 3.81, there are certain limits to what they can do. One of these limits is with parallel building. Parallel building is where a build tool like Make divvys up its work in order to take full advantage of all logical cores (threads) of the CPU. This is one significant way a build can be sped up, especially for larger projects.
```

Comment:
Not sure if “divvys” is clearer here. The macOS dictionary defines it as informal. I generally try to avoid informal/less common words so it’s more likely to be understood by people who aren’t fluent in English.

Reply:
Alright. Reverted it back to "divides its work up."